functions.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * returns an indexed array of associative arrays containing board game information from a database containing said information
+ *
+ * @param PDO $db with link to database containing board game information
+ *
+ * @return array of associative arrays containing individual board game information
+ */
+function getGames (PDO $db) : array
```

Comment:
remove space between function name and the brackets & also the brackets and the colon. (please do this everywhere)